components/core/Helpers/THelper.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+
+namespace AntDesign
+{
+    public static class THelper
```

Comment:
This class can be refactor to generic class and cache the reflect result in memory.